packages/loader/src/resource-deserialize/index.ts

```diff
@@ -24,8 +24,12 @@ export function decode<T>(arrayBuffer: ArrayBuffer, engine: Engine): Promise<T>
   });
 }
 
-export * from "./resources/schema";
-export * from "./resources/scene/SceneParser";
-export * from "./resources/scene/MeshLoader";
-export * from "./resources/scene/EditorTextureLoader";
 export * from "./resources/parser/ParserContext";
+export * from "./resources/scene/EditorTextureLoader";
+export * from "./resources/scene/MeshLoader";
+export * from "./resources/scene/SceneParser";
+export * from "./resources/schema";
+
+export * from "./utils/BufferReader";
+export * from "./utils/Decorator";
+export * from "./utils/FileHeader";
```

Comment:
Why need export these?

Reply:
Because `XRImageReferenceLoader` and `XRImageReferenceDecode` need to parse custom binaries.